src/IniWriter.php

```diff
@@ -119,7 +119,7 @@ public function writeToString(array $config, $header = '')
     private function encodeValue($value)
     {
         if (is_bool($value)) {
-            return (int) $value;
+            return $value ? 'true' : false;
```

Comment:
I guess the `false` would also need to be written as string, so it's converted back correctly. A boolean `false` here will result in an empty value in the config

```suggestion
            return $value ? 'true' : 'false';
```

Reply:
That's right, thank you for spotting that!